app/Http/Controllers/GetPatientInformationController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Resources\PatientResource;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Facades\Auth;
+
+class GetPatientInformationController extends Controller
+{
+    public function __invoke() : JsonResponse
```

Comment:
No deberias agregar un request para validar que tengas info de paciente? si no te va a explotar.

Seguro te falta un test aca, q pasa si user->patient is null